core/src/edu/asu/c3simulator/simulation/Employee.java

```diff
@@ -0,0 +1,259 @@
+		Employee employee = new Employee();
+		employee.employeeName = getRandomEmployeeName();
+		employee.employeePreferredHourlyWage = getPreferredHourlyWage();
+		employee.employeePosition = getEmployeePosition();
+		
+		return employee;
+	}
+	
+	public String getEmployeeName()
+	{
+		return employeeName;
+	}
+	
+	public void setEmployeeName(String employeeName)
+	{
+		this.employeeName = employeeName;
+	}
+	
+	public static String getRandomEmployeeName()
+	{
+		String[] firstName = { "Nicholas", "Colton", "Sawyer", "Frankie", "Jennifer",
+				"Charles", "Kathy", "James", "Jacob", "Crystal", "Mike", "Cody", "Jorge",
+				"Sam", "Lisa", "Margaret", "Marth", "Roy", "Victoria", "Susan", "Ted" };
+		String[] lastName = { "Krogstad", "Tucker", "Hardenbech", "Connelley", "Li",
+				"Baker", "Tyke", "Mattingly", "Feng", "Lee", "Smith", "Houston",
+				"Franco", "White", "Andrade", "Manning", "Brady", "Boyle", "Terry" };
+		
+		int randomFirst = (int) (Math.random() * firstName.length);
+		int randomLast = (int) (Math.random() * lastName.length);
+		
+		String randomEmployeeName = firstName[randomFirst] + " " + lastName[randomLast];
+		
+		return randomEmployeeName;
+	}
+	
+	public void setRandomEmployeeName()
+	{
+		this.employeeName = getRandomEmployeeName();
+	}
+	
+	public static String getEmployeePosition()
+	{
+		String[] positions = { "Manager", "Product Design", "Sales Associate",
+				"Marketing" };
+		int randomPositions = (int) (Math.random() * positions.length);
+		employeePosition = positions[randomPositions];
+		return employeePosition;
+	}
+	
+	public void setEmployeePosition(String employeePosition)
+	{
+		this.employeePosition = employeePosition;
+	}
+	
+	public int getHourlyWage()
+	{
+		return employeePay;
+	}
+	
+	public void setHourlyWage(int employeePay)
+	{
+		this.employeePay = employeePay;
+	}
+	
+	/**
+	 * Preferred Hourly Wage is determined by the selected employee's position and returns
+	 * an integer between the given pay range
+	 */
+	public static int getPreferredHourlyWage()
+	{
+		if (employeePosition == "Manager")
+		{
+			Random r = new Random();
+			int lowPay = 17;
+			int highPay = 24;
+			int i = r.nextInt(highPay - lowPay);
+			Employee.employeePreferredHourlyWage = i + lowPay;
+		}
+		
+		else if (employeePosition == "Product Design")
+		{
+			Random r = new Random();
+			int lowPay = 14;
+			int highPay = 19;
+			int i = r.nextInt(highPay - lowPay);
+			employeePreferredHourlyWage = i + lowPay;
+		}
+		
+		else if (employeePosition == "Sales Associate")
+		{
+			Random r = new Random();
+			int lowPay = 11;
+			int highPay = 16;
+			int i = r.nextInt(highPay - lowPay);
+			employeePreferredHourlyWage = i + lowPay;
+		}
+		
+		else if (employeePosition == "Marketing")
+		{
+			Random r = new Random();
+			int lowPay = 8;
+			int highPay = 13;
+			int i = r.nextInt(highPay - lowPay);
+			employeePreferredHourlyWage = i + lowPay;
+		}
+		
+		return employeePreferredHourlyWage;
+	}
```

Comment:
This has major design flaws. 
Any given instance of employee will have an inconsistent preferredHourlyWage from moment to moment.

A red flag here is the fact that it is static; if it should be static, please add a doc explaining why. Otherwise, this needs fixing